sdk/keyvault/tests.yml

```diff
@@ -1,5 +1,7 @@
-variables:
-  ProjectFile: sdk/keyvault/Microsoft.Azure.KeyVault.sln
+trigger: none
 
 jobs:
-- template: ../../eng/pipelines/templates/jobs/tests.yml
\ No newline at end of file
+- template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+  parameters:
+    MaxParallel: 1
+    ServiceDirectory: keyvault
```

Comment:
How would env variables be managed per project? Do we need to make a distinction to know which ones are used for track 1 and which ones for track2?
It looks like you add `EnvVars` to the section and then the variables. in a generic way. Names can always be different and this will help, but just wondering

Reply:
Environment variables are configured with the `EnvVars` parameter like in this example: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/tests.yml

Reply:
Yes, but now that there is only one test.yml file for both solutions (track1, track2) I wonder how do we distinguish which variables are used for which project

Reply:
Talked offline and yes, we need to make sure that the names selected for the env variables won't cause issues with other existing solutions in the folder/scope of the `test.yml`